Steps/PublishCodeCoverage.yml

```diff
@@ -10,13 +10,31 @@ parameters:
   displayName: Perform self checkout step
   type: boolean
   default: true
+- name: setup_python
+  displayName: Setup Python
+  type: boolean
+  default: true
+- name: reorganize_by_inf

```

Comment:
Like the other file, it would be nice to keep the `type` on each parameter explicit for consistency.

Reply:
Added typing to reorganize_by_inf and build_file parameters as requested. Thanks!